src/views/BatchInfoPage/BatchInfoPageContent.tsx

```diff
@@ -7,6 +7,7 @@ import '../../scss/batch-info-page-content.scss';
 import { IUser } from '../../_reducers/UserReducer';
 import BatchCardLarge from '../../components/BatchCardLarge/BatchCardLarge';
 import { AssociateCard } from '../../components/AssociateCard/AssociateCard';
+import Notifications from '../../components/Notifications/Notifications';
 
 interface IBatchInfoPageContentProps {
```

Comment:
Not exclusive to batch info, but there are inconsistencies in where we store interfaces. 1/2-2/3 of all interfaces are currently being stored in the ./_reducers path.

Reply:
yea. if i used an interface that already existed, i left it where it was. if its a new one, i separated it out into a file that should hold most of them. I agree they all should be in one place. I'm pretty sure @nickBarak will consolidate them all, if he hasn't already.